src/ast/select.rs

```diff
@@ -101,11 +101,35 @@ impl<'a> Select<'a> {
         T: Into<Table<'a>>,
     {
         Select {
-            table: Some(Box::new(table.into())),
+            tables: vec![Box::new(table.into())],
             ..Default::default()
         }
     }
 
+    /// Adds a table to be selected.
+    ///
+    /// ```rust
+    /// # use quaint::{ast::*, visitor::{Visitor, Sqlite}};
+    /// # fn main() -> Result<(), quaint::error::Error> {
+    /// let query = Select::from_table("users")
+    ///     .from(Table::from(Select::default().value(1)).alias("num"))
+    ///     .column(("users", "name"))
+    ///     .value(Table::from("num").asterisk());
+    ///
+    /// let (sql, _) = Sqlite::build(query)?;
+    ///
+    /// assert_eq!("SELECT `users`.`name`, `num`.* FROM `users`, (SELECT ?) AS `num`", sql);
+    /// # Ok(())
+    /// # }
+    /// ```
+    pub fn from<T>(mut self, table: T) -> Self
```

Comment:
Name this `and_from`, `from` is reserved for `From` trait.

Reply:
Done

Reply:
Actually, now when I think about this, can this be anything else than another `Select`? If so, please change the signature into `T: Into<Select>`...

Reply:
And if this always needs to have an alias, then this would be:

```rust
pub fn and_from(mut self, select: impl Into<Select>, alias: impl Into<Cow<'a, str>>) -> Self {
    ..
}
```